build/cloud/07-deployment.yaml

```diff
@@ -1,57 +0,0 @@
-apiVersion: apps/v1
```

Comment:
no deployment.yaml for pre1.3?

Reply:
@kevin-wangzefeng I'm sorry that maybe it's deleted by mistake, I will add it now.